### PR TITLE
4 search bar functionality

### PR DESCRIPTION
**Pull request by Flam64**

### CODE REVIEW

**Review by Aurel1H — DISMISSED** [review]
Search bar OK

**Review by ChristineV67 — DISMISSED** [review]

**Review by RomainVarra — APPROVED** [review]

File: client/src/pages/SearchPage.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import ViewSearchList from "../components/ViewSearchList";
+
+export default function SearchPage() {
+  return (
+    <>
+      <ViewSearchList />
+
+      {/* Call component ViewSearch */}
```
Comment (ioayoub) [inline client/src/pages/SearchPage.tsx:8]: Dead code

File: client/src/main.tsx [hunk 2]
```diff
@@ -7,6 +7,10 @@ import { RouterProvider, createBrowserRouter } from "react-router-dom";
 
 // Import the main app component
```
Comment (ioayoub) [inline client/src/main.tsx:8]: Dead code

File: client/src/main.tsx [hunk 3]
```diff
@@ -7,6 +7,10 @@ import { RouterProvider, createBrowserRouter } from "react-router-dom";
...
 import App from "./App";
+//import AlbumsPage from "./pages/AlbumsPage";
```
Comment (ioayoub) [inline client/src/main.tsx:10]: Dead code

File: client/src/main.tsx [hunk 4]
```diff
@@ -7,6 +7,10 @@ import { RouterProvider, createBrowserRouter } from "react-router-dom";
...
+import HomePage from "./pages/HomePage";
+//import ArtistPage from "./pages/ArtistPage";
```
Comment (ioayoub) [inline client/src/main.tsx:12]: Dead code

File: client/src/components/search.css [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+  height: 30px;
+  border-radius: 3px;
+}
+
+.search {
+  padding: 0.28rem;
+  margin: auto;
+  width: fit-content;
+  background-color: var(--prymary-color-bg);
+  /* border-radius: 0 10px 10px 0; */
```
Comment (ioayoub) [inline client/src/components/search.css:26]: Dead code

File: client/src/components/NavBar.tsx [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+    <>
+      <div className="navBarMW">
+        <img
+          className="logoMW"
+          src="../../public/Images/logo-musiwild-without-text.png"
+          alt="logo du site MusicWild"
+        />
+        <img
+          className="titleMW"
+          src="../../public/Images/MUSICWILD.png"
```
Comment (ioayoub) [inline client/src/components/NavBar.tsx:15]: public folder is accessible by your web app, you can update the import to "/Images/MUSICWILD.png"

File: client/src/components/NavBar.tsx [hunk 7]
```diff
@@ -0,0 +1,40 @@
+import { NavLink } from "react-router-dom";
+import Search from "./Search";
+
+function NavBar() {
+  return (
+    <>
+      <div className="navBarMW">
+        <img
+          className="logoMW"
+          src="../../public/Images/logo-musiwild-without-text.png"
```
Comment (ioayoub) [inline client/src/components/NavBar.tsx:10]: public folder is accessible by your web app, you can update the import to "/Images/logo-musiwild-without-text.png"

File: client/src/components/Albums.tsx [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+          </div>
+        ))}
+      </div>
+    </article>
+  );
+}
+
+export default Albums;
+
+/* <p>{m.genre}</p>
```
Comment (ioayoub) [inline client/src/components/Albums.tsx:41]: This 👀

**Review by ioayoub — CHANGES_REQUESTED** [review]

File: client/src/pages/SearchPage.tsx (hunk 1 above)
Comment (Aurel1H) [reply client/src/pages/SearchPage.tsx]: Dead code deleted

File: client/src/main.tsx (hunk 2 above)
Comment (Aurel1H) [reply client/src/main.tsx:8]: Dead code deleted

File: client/src/main.tsx (hunk 3 above)
Comment (Aurel1H) [reply client/src/main.tsx]: Dead code deleted

File: client/src/main.tsx (hunk 4 above)
Comment (Aurel1H) [reply client/src/main.tsx]: Dead code deleted

File: client/src/components/search.css (hunk 5 above)
Comment (Aurel1H) [reply client/src/components/search.css]: Dead code deleted

File: client/src/components/NavBar.tsx (hunk 7 above)
Comment (Aurel1H) [reply client/src/components/NavBar.tsx]: Dead code deleted

File: client/src/components/Albums.tsx (hunk 8 above)
Comment (Aurel1H) [reply client/src/components/Albums.tsx]: Dead code deleted

File: client/src/components/NavBar.tsx (hunk 6 above)
Comment (Aurel1H) [reply client/src/components/NavBar.tsx]: Dead code deleted

**Review by ioayoub — APPROVED** [review]

**Review by Aurel1H — APPROVED** [review]

**Review by ChristineV67 — APPROVED** [review]

**Review by RomainVarra — APPROVED** [review]

**Pull request merged**